src/main/java/teammates/common/util/Templates.java

```diff
@@ -48,6 +48,14 @@ public static String populateTemplate(String template, String... keyValuePairs)
                 FileHelper.readResourceFile("userEmailTemplate-feedbackSessionPublished.html");
         public static final String USER_FEEDBACK_SUBMISSION_CONFIRMATION =
                 FileHelper.readResourceFile("userEmailTemplate-feedbackSubmissionConfirmation.html");
+        public static final String FRAGMENT_SESSION_LINK_RECOVERY_ACCESS_LINKS =
```

Comment:
SESSION_LINKS*?

Reply:
This fragment is actually just for one session link